infra.bs

```diff
@@ -327,17 +327,22 @@ is immaterial. <a>Strings</a> are denoted by double quotes and monospace font.
 <p class=example id=example-string-notation>"<code>Hello, world!</code>" is a string.
 
 <p>To <dfn export for="JavaScript string">convert</dfn> a <a>JavaScript string</a> into a
-<a>scalar value string</a>, replace any <a>surrogate code points</a> with U+FFFD.
-<span class=note>Per definition these are isolated surrogates.</span>
+<a>scalar value string</a>, replace any <a>surrogates</a> with U+FFFD.
 <!-- Obviates need for https://heycam.github.io/webidl/#dfn-obtain-unicode -->
 
-<p class=note>In documents using the Infra Standard, a <a>scalar value string</a> can always be used
-as <a>JavaScript string</a> implicitly since it is a subset. The reverse is only possible if the
-<a>JavaScript string</a> is known to not contain <a>surrogates</a>. (An implementation likely has to
-perform explicit conversion, depending on how it actually ends up representing
-<a lt="JavaScript string">JavaScript</a> and <a>scalar value strings</a>. It is even fairly typical
-for implementations to have multiple implementations of just <a>JavaScript strings</a> for
-performance reasons and reducing memory usage.)
+<p class=note>The replaced surrogates are always isolated surrogates, since the process of
+interpreting the JavaScript string as containing <a>code points</a> will have converted surrogate
+pairs into single non-surrogate code points.)
+
+A <a>scalar value string</a> can always be used as <a>JavaScript string</a> implicitly since it is a
```

Comment:
Missing `<p>`. I'm surprised Bikeshed didn't complain to me about "surrogate code point" above.